model.js

```diff
@@ -23,9 +23,6 @@ function Users() {
         return users.push(user);
     };
     //      //TODO Надо ли оно

```

Comment:
It is front-end side model.
Here model should be. Here should be setter for the whole users list. The setter should be called at the application initialization.